tests/integration_tests/build/test_binary_size.py

```diff
@@ -11,13 +11,13 @@
 MACHINE = platform.machine()
 """ Platform definition used to select the correct size target"""
 
-FC_BINARY_SIZE_TARGET = 3124000 if MACHINE == "x86_64" else 3349336
+FC_BINARY_SIZE_TARGET = 3013464 if MACHINE == "x86_64" else 3251464
 """Firecracker target binary size in bytes"""
 
-FC_BINARY_SIZE_LIMIT = 3300000 if MACHINE == "x86_64" else 3500000
+FC_BINARY_SIZE_LIMIT = 32000000 if MACHINE == "x86_64" else 3400000
 """Firecracker maximum binary size in bytes"""
 
-JAILER_BINARY_SIZE_TARGET = 2483592 if MACHINE == "x86_64" else 2714528
+JAILER_BINARY_SIZE_TARGET = 2568384 if MACHINE == "x86_64" else 2792688
```

Comment:
No, it is not related. But these values weren't updated for a long time (or maybe never) and I thought it could be a good time to do that (since the others were updated recently with the rust update).

We might want to consider though either a tighter tolerance or a smaller limit for the jailer since this binary increased pretty much lately and this test didn't fail :-?.